plugin/src/main/kotlin/tanvd/grazi/spellcheck/GraziSpellchecker.kt

```diff
@@ -77,7 +85,12 @@ object GraziSpellchecker : GraziStateLifecycle {
      * Note, that casing and plural typos are ignored.
      */
     private fun check(word: String, project: Project, language: Language) = buildSet<Typo> {
-        val typo = tryRun { checker.check(word) }?.firstOrNull()?.let { Typo(it, checkerLang, 0) }
+        val typo = try {
+            checker.check(word)
+        } catch (e: Throwable) {
+            logger.trace("GraziSpellchecker exception", e)
```

Comment:
`Exception occurred during check in GraziSpellchecker` ;-)

Reply:
Fixed